lib/ansible/modules/windows/win_domain_object_info.ps1

```diff
@@ -0,0 +1,99 @@
+#!powershell
+
+# Copyright: (c) 2019, Shachaf Goldstein <shachaf.gold@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+#AnsibleRequires -CSharpUtil Ansible.Basic
+
+Set-StrictMode -Version 2.0
+
+$spec = @{
```

Comment:
For each of these entries, `required = $false` is the default and can just be omitted to simplify it all.